### DISCUSSION TITLE
Kiosk App Bypass (Open kiosk apps in normal browser)

### DISCUSSION THREAD

**Discussion by Blobby-Boi** [3kh0/ext-remover#1009] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
DISCLAIMER: CHEATING WITH THIS CAN GET YOU INTO LOT OF TROUBLE. DONT ABUSE THIS TOO MUCH!
Find your kiosk app in this list:

TestNav
https://sites.google.com/view/test-nav/

Secure Test Browser
https://sites.google.com/view/securetestbrowser

Riverside DataManager
https://sites.google.com/view/riverside-datamanager/

Illuminate Secure Browser
https://sites.google.com/view/illuminate-secure-browser/

If you want me to add one just let me know!

**Comment by v4pl** [comment]
Nice exploit

**Reply by Blobby-Boi** [reply]
Thx

**Comment by bananstache** [comment]
is this legit?

**Reply by Blobby-Boi** [reply]
Ye

**Reply by bananstache** [reply]
Why not just go to the official site??
TestNav: https://home.testnav.com/
Secure Test Browser: https://login5.cambiumtds.com/student_core/V74/Pages/LoginShell.aspx?c=California_PT&a=Student
Riverside Data Manager: https://www.riversidedatamanager.com/BalancedManagement/user/signin
I couldnt find one for illuminate :/

**Comment by xoidex** [comment]
just to confirm; these are kiosk apps in the normal browser right

**Reply by Blobby-Boi** [reply]
yeah

**Reply by xoidex** [reply]
ty title was confusing

also could you add Questar NY?

**Reply by Blobby-Boi** [reply]
Ok I’ll try

**Reply by xoidex** [reply]
thanks

**Comment by AshtonDavies** [comment]
It is insane that this works, but cheating with it is not a good idea. You could get in serious trouble, so be careful if you decide to do this for kiosk exams.

You could, however, use this if your Chromebook is unenrolled with an exploit and you need a way to test on it. Just have to use fullscreen mode and you should blend in fine.

**Reply by Blobby-Boi** [reply]
I actually was not planning on cheating with this. I just want it to be less annoying since I don’t wanna sign out every time. I should probably add a disclaimer to it though

**Reply by AshtonDavies** [reply]
> I actually was not planning on cheating with this. I just want it to be less annoying since I don’t wanna sign out every time. I should probably add a disclaimer to it though

That's valid actually. You don't have to save your work and sign out every time with this.

**Comment by AshtonDavies** [comment]
I have noticed a user on the Internet using a Chrome OS kiosk app on Windows, for anyone who is interested.

I was going to share the link, but I do not remember what the YouTube video was titled.

**Reply by Blobby-Boi** [reply]
Was it the actual kiosk app? Or was it like this where it’s a url that opens the kiosk app

**Reply by AshtonDavies** [reply]
> Was it the actual kiosk app? Or was it like this where it’s a url that opens the kiosk app

It was an installed application on their desktop, maybe through Chrome if they got the CRX for it and sideloaded the regular Chrome app that wasn't meant to be used that way.

**Reply by Blobby-Boi** [reply]
Dats cool… Im gonna try to see if I can get it to run on mine then :P

**Reply by Blobby-Boi** [reply]
It just won’t install from the webstore

**Reply by schoolexploitkid** [reply]
its in tn

**Comment by NuggetKid35** [comment]
It doesn't work anymore can u do a new one fast I have a test coming on kiosk4school

**Reply by Blobby-Boi** [reply]
why are you replying on this year-and-a-half old post 😭
I lwk don't even remember what this was I think I deleted the sites at some point cuz they stopped working

**Reply by NuggetKid35** [reply]
It's called Kiosk4School, and it's an app on Chromebooks designed to prevent cheating in schools. You provided website links website links where u didn't have to use the app, but those links no longer work. I'm just asking if you could make those sites functional again, as they would serve as the ultimate cheating tool.

**Reply by S-PScripts** [reply]
> It's called Kiosk4School, and it's an app on Chromebooks designed to prevent cheating in schools. You provided website links website links where u didn't have to use the app, but those links no longer work. I'm just asking if you could make those sites functional again, as they would serve as the ultimate cheating tool.

invent it yourself

**Reply by NuggetKid35** [reply]
> > It's called Kiosk4School, and it's an app on Chromebooks designed to prevent cheating in schools. You provided website links website links where u didn't have to use the app, but those links no longer work. I'm just asking if you could make those sites functional again, as they would serve as the ultimate cheating tool.
> 
> invent it yourself

Did I ask you something just stfu bro

**Reply by NuggetKid35** [reply]
> why are you replying on this year-and-a-half old post 😭 I lwk don't even remember what this was I think I deleted the sites at some point cuz they stopped working

Ofc bro closed this laziness on its finest isn't it?

**Reply by S-PScripts** [reply]
😂

**Reply by xoidex** [reply]
dude this guy is actually so entitled lmao

**Reply by S-PScripts** [reply]
> dude this guy is actually so entitled lmao

ok pal

**Comment by ghost** [comment]
404 error on Google site
